### PR TITLE
Adds ERP

### PR DESCRIPTION
**Pull request by TheNeoGamer42**
## About The Pull Request
adds a perfectly normal emote, *sex
## Why It's Good For The Game
sex good
## Changelog
:cl:
add: an ERP emote, *sex
/:cl:

### CODE REVIEW

**Comment by Superyodeler** [comment]
It's even modularized properly

File: waspstation/code/modules/mob/emote.dm [hunk 1]
```diff
@@ -24,3 +24,14 @@
...
+/datum/emote/sex
+	key = "sex"
+	key_third_person = "DOES AN ILLEGAL"
+	restraint_check = TRUE
+	mob_type_allowed_typecache = list(/mob/living)
+
+/datum/emote/sex/run_emote(mob/user, params, type_override, intentional)
+	. = ..()
+	if(.)
+		user.gib()
```
Comment (dinnerwoerror) [inline waspstation/code/modules/mob/emote.dm:37]: if(.)
user.gib()

**Comment by TheNeoGamer42** [comment]
I take pride in adding the best feature to the game, why wouldn't I modularize it?

File: waspstation/code/modules/mob/emote.dm (hunk 1 above)
Comment (TheNeoGamer42) [reply waspstation/code/modules/mob/emote.dm:37]: I believe that is how one has sex, right?

**Comment by TheNeoGamer42** [comment]
There is now an appropriate sound for *sex

**Comment by TheNeoGamer42** [comment]
I wonder if this even works still

**Review by Shadowtail117 — CHANGES_REQUESTED** [review]
The modularization folder is no longer `waspstation`, it's `whitesands`.

**Comment by TheNeoGamer42** [comment]
true, idfk how to change it without reinstalling it and shit though

**Comment by Shadowtail117** [comment]
Given your last commit was June 10th of last year, the branch is likely way too outdated to reasonably salvage. I recommend starting with a new branch that's been fast-forwarded and carrying your changes over to there in a new PR.

**Comment by TheRealScarHomie** [comment]
remember to tactically sex in near death situations, your brain can be salvaged easier than your body

**Pull request closed without merging**